src/resources/review/review.model.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const ReviewSchema = new Schema(
+  {
+    review: {
```

Comment:
text message should be required as we do not want to store empty comments
same for name. user without name or anonymous comments should not be allowed